example/src/screens/BasicFullscreenHandling.tsx

```diff
@@ -8,7 +8,6 @@ import {
   AudioSession,
```

Comment:
I reworked the sample to make the player view actually full screen by hiding the navigation bar.